plugins/local_file/CMakeLists.txt

```
@@ -0,0 +1,51 @@
+add_custom_target(all-plugins-local_file)
+add_dependencies(all-plugins all-plugins-local_file)
+
+set(
+  IRODS_EXPERIMENTAL_API_PLUGINS
+  main
+)
+
+foreach(plugin IN LISTS IRODS_EXPERIMENTAL_API_PLUGINS)
+  string(TOUPPER ${plugin} PLUGIN_UPPERCASE)
+  set(plugin_target "irods_http_plugin-${plugin}")
+  set(plugin_name "${plugin}")
+
+  add_library(
+    ${plugin_target}
+    MODULE
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/${plugin}.cpp"
+  )
+  set_property(TARGET ${plugin_target} PROPERTY LIBRARY_OUTPUT_NAME ${plugin_name})
+  target_link_libraries(
+    ${plugin_target}
+    PRIVATE
+    irods_common
+    "${IRODS_EXTERNALS_FULLPATH_BOOST}/lib/libboost_filesystem.so"
+    "${IRODS_EXTERNALS_FULLPATH_BOOST}/lib/libboost_system.so"
+    fmt::fmt
+    ${CMAKE_DL_LIBS}
+  )
+  target_include_directories(
+    ${plugin_target}
+    PRIVATE
+    "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+    "${IRODS_EXTERNALS_FULLPATH_BOOST}/include"
+  )
+  target_compile_definitions(
+    ${plugin_target}
+    PRIVATE
+    ${IRODS_COMPILE_DEFINITIONS_PRIVATE}
+    ENABLE_RE
+    IRODS_ENABLE_SYSLOG
+  )
+
+  add_dependencies(all-plugins-local_file ${plugin_target})
```

Let's get rid of `all-plugins-local_file` and just add the plugin target directly to `all-plugins`'s dependencies.

```suggestion
  add_dependencies(all-plugins ${plugin_target})
```

---

Should be unneeded as 'all-plugins' target is removed.